src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -146,100 +198,223 @@
       "The timezone to use when partitioning with ``TimeBasedPartitioner``.";
   public static final String TIMEZONE_DEFAULT = "";
+  private static final String TIMEZONE_DISPLAY = "Timezone";
 
   public static final String FILENAME_OFFSET_ZERO_PAD_WIDTH_CONFIG = "filename.offset.zero.pad.width";
   private static final String FILENAME_OFFSET_ZERO_PAD_WIDTH_DOC =
       "Width to zero pad offsets in HDFS filenames to if the offsets is too short in order to "
       + "provide fixed width filenames that can be ordered by simple lexicographic sorting.";
   public static final int FILENAME_OFFSET_ZERO_PAD_WIDTH_DEFAULT = 10;
+  private static final String FILENAME_OFFSET_ZERO_PAD_WIDTH_DISPLAY = "Filename Offset Zero Pad Width";
 
-  public static final String HDFS_AUTHENTICATION_KERBEROS_CONFIG = "hdfs.authentication.kerberos";
-  private static final String HDFS_AUTHENTICATION_KERBEROS_DOC =
-      "Configuration indicating whether HDFS is using Kerberos for authentication.";
-  private static boolean HDFS_AUTHENTICATION_KERBEROS_DEFAULT = false;
+  // Schema group
+  public static final String SCHEMA_COMPATIBILITY_CONFIG = "schema.compatibility";
+  private static final String SCHEMA_COMPATIBILITY_DOC =
+      "The schema compatibility rule to use when the connector is observing schema changes. The "
+      + "supported configurations are NONE, BACKWARD, FORWARD and FULL.";
+  private static final String SCHEMA_COMPATIBILITY_DEFAULT = "NONE";
+  private static final String SCHEMA_COMPATIBILITY_DISPLAY = "Schema Compatibility";
 
-  public static final String CONNECT_HDFS_PRINCIPAL_CONFIG = "connect.hdfs.principal";
-  private static final String CONNECT_HDFS_PRINCIPAL_DOC =
-      "The principal to use when HDFS is using Kerberos to for authentication.";
-  public static final String CONNECT_HDFS_PRINCIPAL_DEFAULT = "";
+  public static final String SCHEMA_CACHE_SIZE_CONFIG = "schema.cache.size";
+  private static final String SCHEMA_CACHE_SIZE_DOC =
+      "The size of the schema cache used in the Avro converter.";
+  public static final int SCHEMA_CACHE_SIZE_DEFAULT = 1000;
+  private static final String SCHEMA_CACHE_SIZE_DISPLAY = "Schema Cache Size";
 
-  public static final String CONNECT_HDFS_KEYTAB_CONFIG = "connect.hdfs.keytab";
-  private static final String CONNECT_HDFS_KEYTAB_DOC =
-      "The path to the keytab file for the HDFS connector principal. "
-      + "This keytab file should only be readable by the connector user.";
-  public static final String CONNECT_HDFS_KEYTAB_DEFAULT = "";
+  // Internal group
+  public static final String STORAGE_CLASS_CONFIG = "storage.class";
+  private static final String STORAGE_CLASS_DOC =
+      "The underlying storage layer. The default is HDFS";
+  public static final String STORAGE_CLASS_DEFAULT = "io.confluent.connect.hdfs.storage.HdfsStorage";
+  private static final String STORAGE_CLASS_DISPLAY = "Storage Class";
+
+  private static final String HDFS_GROUP = "hdfs";
```

Comment:
Should the groups be capitalized? We probably want them to make sense as titles of sections of the UI, and some aren't obvious how to auto-capitalize (like HDFS).